crates/swc_ecma_minifier/tests/fixture/issues/4845/output.js

```diff
@@ -0,0 +1 @@
+console.log(a + (b ? "c" : "d") + "1");
```

Comment:
It's wrong to blindly turn number into string in string context, it would generate an extra "", but this is well beyond this pr's scope